### PR TITLE
Add notes field to Scenario

### PR DESCRIPTION
**Pull request by elsiel23**
Unintentionally includes changes from #553 that were merged into this branch instead of main.

### CODE REVIEW

**Comment by elsiel23** [comment]
@lattemilk FYI

File: src/planscape/plan/models.py [hunk 1]
```diff
@@ -88,6 +88,8 @@ class Scenario(models.Model):
 
     project = models.ForeignKey(
         Project, on_delete=models.CASCADE, null=True)  # type: ignore
+    
+    notes: models.TextField = models.TextField(null=True)
```
Comment (strangeties) [inline src/planscape/plan/models.py:92]: nit: can the assignment format be consistent in this file? Some of these fields have type hints while others don't.

**Comment by strangeties** [comment]
Looks mostly fine! Can you please update the pr name and comment to show that this
1) adds a notes field to scenario and
2) adds a get_scenario endpoint?

**Review by strangeties — APPROVED** [review]

File: src/planscape/plan/models.py (hunk 1 above)
Comment (elsiel23) [reply src/planscape/plan/models.py:92]: I think this is intentional, they are only assigned for FK fields.

**Pull request merged**